play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukCookieBanner.scala.html

```diff
@@ -15,12 +15,14 @@
  *@
 
 @this(govukButton: GovukButton)
-
 @(params: CookieBanner)
 @import params._
 
-<div class="@toClasses("govuk-cookie-banner ", classes)" data-nosnippet role="region" aria-label="@ariaLabel.getOrElse("Cookie banner")"@if(hidden){ hidden}@toAttributes(attributes)>@for(message <- messages) {
-    <div class="@toClasses("govuk-cookie-banner__message govuk-width-container", message.classes)"@message.role.mapNonEmpty {role => role="@role"} @toAttributes(message.attributes) @if(message.hidden){ hidden}>
+<div class="@toClasses("govuk-cookie-banner", classes)" data-nosnippet role="region" aria-label="@{ariaLabel match {
+    case Some(NonEmptyString(str)) => str
+    case _ => "Cookie banner"
+   }}"@if(hidden){ hidden}@toAttributes(attributes)>@for(message <- messages) {
+    <div class="@toClasses("govuk-cookie-banner__message", message.classes, "govuk-width-container")"@message.role.mapNonEmpty {role => role="@role"} @toAttributes(message.attributes) @if(message.hidden){ hidden}>
```

Comment:
guessing fixtures would catch if it wasn't right but 
```
@message.role.mapNonEmpty {role => role="@role"}
```
seems really mind bending to me too - in my head I was expecting `{role => role="@role"}` to be scala rather than a twirl block because I read @ in this case as output variable, so you'd need to be building a string - #twirlthings 😵

Reply:
in my head it's difference between interpretting stuff as a twirl statement
```
@if(message.role.nonEmpty) { role="@message.role"}
```
vs inline scala
```
@{if (message.role.nonEmpty) s"""role="$message.role"""" else ""}
```
guess my intuition is getting where it's `@var` a bit wrong - want like a twirl cheat sheet / test harness with table of "this renders as that"